tools/topology/topology1/CMakeLists.txt

```diff
@@ -248,6 +248,8 @@ set(TPLGS
 	"sof-glk-es8336\;sof-tgl-es8336-dmic4ch-ssp1\;-DPLATFORM=tgl\;-DSSP_NUM=1\;-DCHANNELS=4"
 	"sof-glk-es8336\;sof-tgl-es8336-dmic4ch-ssp2\;-DPLATFORM=tgl\;-DSSP_NUM=2\;-DCHANNELS=4"
 
+	"sof-glk-es8336\;sof-adl-es83x6-ssp1\;-DPLATFORM=adl\;-DSSP_NUM=1\;-DCHANNELS=0"
```

Comment:
no, please follow the same convention as tgl and all other platforms. I don't see why you should start using 8x36 when all other topologies use 8336.

Also please add all possible variants so that we are done for ADL.

Reply:
As We had tested this topology with ES8326 codec, we made the topology name as generic. Also as we had made the topology name as generic in  our driver code(https://github.com/thesofproject/linux/pull/3736/ => soc-acpi-intel-adl-match.c). Please let us know if we need any further changes on driver side as well.

Reply:
The `915` GPU was released in 2004. The `i915` driver is still being used for today's GPUs. No one died :-)
https://github.com/thesofproject/linux/blame/topic/sof-dev/drivers/gpu/drm/i915/display/intel_dmc.c

Reply:
We have made changes as suggested here. Also, we will submit a fixup for the soc-acpi table.